src/components/Loginn/TokenValid.js

```diff
@@ -0,0 +1,33 @@
+import { useNavigate } from 'react-router-dom';
+import {  useEffect } from 'react';
+
+
+
+function TokenValid() {
```

Comment:
Also I don't quite understand why we need new component for that,
We have two forms for user login and registration, both forms return accessToken in case of successful authorization, in that case, we need to save the token from response, not in separate component